components/status-indicator/test/status-indicator.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+	<title>d2l-status-indicator unit tests</title>
+	<script src="/node_modules/@webstatusIndicators/webstatusIndicatorsjs/webstatusIndicators-loader.js"></script>
+	<script src="/node_modules/mocha/mocha.js"></script>
+	<script src="/node_modules/chai/chai.js"></script>
+	<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+	<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
+	<script type="module" src="../status-indicator.js"></script>
+</head>
+
+<body>
+
+	<test-fixture id="subtle">
+		<template>
+			<d2l-status-indicator state="default" text="test subtle"></d2l-status-indicator>
+		</template>
+	</test-fixture>
+
+	<test-fixture id="bold">
+		<template>
+			<d2l-status-indicator state="default" text="test bold" bold></d2l-status-indicator>
+		</template>
+	</test-fixture>
+
+	<script type="module">
+		import { runAxe } from '../../../tools/a11y-test-helper.js';
+
+		describe('d2l-status-indicator', () => {
+
+			let statusIndicator;
+
+			describe('subtle', () => {
+
+				beforeEach(async() => {
+					statusIndicator = fixture('subtle');
+					await statusIndicator.updateComplete;
+				});
+
+				it('passes all axe tests', async() => {
+					await runAxe(statusIndicator);

```

Comment:
I think we'll want to run the aXe tests for each possible state + bold value, just to make sure all the colour contrast passes. Should be able to do it in a single test though using some fancy `forEach`-ing.